components/site/docsLayout.tsx

```diff
@@ -0,0 +1,13 @@
+import Layout from './layout';
+import DocsNavigation from './docsNavigation';
+
+type LayoutProps = {
+  children?: React.ReactNode;
+};
+
+const DocsLayout = (props: LayoutProps): JSX.Element => {
+  const { children } = props;
+  return <Layout navigationComponent={<DocsNavigation />}>{children}</Layout>;
```

Comment:
Technically, don't really need this component at all and could just add this code to the `[...slug]` page